neural_lam/datastore/base.py

```diff
@@ -455,6 +472,9 @@ class BaseRegularGridDatastore(BaseDatastore):
     - `get_xy` (method): Return the x, y coordinates of the dataset, with the
       option to not stack the coordinates (so that they are returned as a 2D
       grid).
+        - `get_lat_lon` (method): Return the latitude/longitude coordinates of
+          the
+            dataset for convenience when plotting.
```

Comment:
Just some weird indentation
```suggestion
    - `get_lat_lon` (method): Return the latitude/longitude coordinates of
      the dataset for convenience when plotting.
```